documentation/breaking-changes/upcoming-breaking-changes-desktopvirtualization.md

```diff
@@ -0,0 +1,78 @@
+# Upcoming breaking changes in Azure PowerShell
+
+## Az.DesktopVirtualization
+
+The following cmdlets were affected this release:
```

Comment:
Are these cause by API version update? Can we be more specific?

Reply:
> Are these cause by API version update? Can we be more specific?

Talked to xiaogang and comfirm that all of them are caused by V4 upgrading